docs/content/Deployment/Platform-Serverless-AWS.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Deploy with Serverless Framework on AWS
+permalink: /deployment/platforms/serverless/aws
+category: Deployment
+subCategory: Platforms
+menuOrder: 2
+---
+
+This guide walks you through deploying Cube.js with the [Serverless
+Framework][link-sls] on [AWS][link-aws].
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| This is an example of a production-ready deployment, but real-world
+| deployments can vary significantly depending on desired performance
+| and scale.
+<!-- prettier-ignore-end -->
+
+## Prerequisites
+
+- An AWS account
+- An [Elasticache Redis][aws-redis] cluster URL
+- A separate Cube Store or external database
```

Comment:
does it make sense to say "an external database for pre-aggregations"?

Reply:
@rpaik @hassankhan I'd remove mentions of the external database since we are planning to deprecate storing pre-aggregations anywhere except the cube store